src/main/java/ua/edu/internship/interview/config/SwaggerConfig.java

```diff
@@ -0,0 +1,28 @@
+package ua.edu.internship.interview.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.servers.Server;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
+public class SwaggerConfig {
+    @Value("${project.version}")
+    private String version;
+    @Value("${domain.back}")
+    private String devServer;
+
+    @Bean
+    public OpenAPI openAPI() {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'openAPI' must contain no more than '2' consecutive capital letters.
<!-- __reviewdog__:ChBlMjU4MWE4ZjRjZDQ1Yzg5EgpjaGVja3N0eWxl -->